client/landing/subscriptions/components/add-sites-button/add-sites-button.tsx

```diff
@@ -0,0 +1,34 @@
+import { isEnabled } from '@automattic/calypso-config';
+import { Button } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { useState } from 'react';
+import './styles.scss';
+import { AddSitesModal } from 'calypso/landing/subscriptions/components/add-sites-modal';
+
+const AddSitesButton = () => {
+	const translate = useTranslate();
+	const [ isAddSitesModalVisible, setIsAddSitesModalVisible ] = useState( false );
+
+	if ( ! isEnabled( 'subscription-management-add-sites-modal' ) ) {
+		return null;
+	}
+
+	return (
+		<>
+			<Button
+				primary
+				className="subscriptions-add-sites-button"
+				onClick={ () => setIsAddSitesModalVisible( true ) }
+			>
+				{ translate( 'Add sites' ) }
```

Comment:
Just a note (so we won't forget): If we agree to display form for one site only, this will likely need to change to `Add site` (and similarly in the modal's title).

Reply:
I think it can stay like it is, because when uploading an OPML file, you're still adding multiple sites.

Reply:
Good point. 👍🏼